src/components/dialog/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { DialogOverlay, DialogContent } from '@reach/dialog'
```

Comment:
I chose to use [reach dialog](https://reach.tech/dialog/) here to go with previous convention. This is a wrapper for the dialog that sets custom base styles.